src/index.html

```diff
@@ -6,7 +6,44 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
-  </body>
+  <div class="calendar">
+    <div class="calendar__start-day-wed"></div>
+    <div class="calendar__month-length-29"></div>
+
+    <div class="calendar__sheet" >
```

Comment:
Not sure that it's a good practice to split the modifiers into elements of the main block

You may reference check-list (p.2)